src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGDropZone.cs

```diff
@@ -208,6 +208,25 @@ public bool IsEmpty()
             return _emptyStatePrefabInstance != null;
         }
 
+        /**
+         * <summary>
+         * Show or hide the empty state assigned to this drop zone
+         * </summary>
+         * <param name="isEmpty">If the empty state should be shown or hidden</param>
```

Comment:
Mega nit pick: The parameter name in conjunction with the param doc confused me, as it describes how the boolean is going to be acted on instead of what it is, I think? Maybe instead of isEmpty it also is called setEmpty but this is splitting hairs at this point, or maybe I misunderstood

Reply:
Good call, a hair worth splitting. I was feeling a bit confused as well, I've changed it to `setEmpty` 👍🏻